### ISSUE TITLE
Get tasks and targets for WPF ported so that projects can built with the .NET Core SDK

### ISSUE THREAD

**Issue by nguerrera** [dotnet/cli#10083]

**Comment by nguerrera** [comment]
There are two parts to this. The estimate of 5 days applies to the part that I will be doing (targets + packaging into WPF SDK) and there is also the porting of the tasks dll that Ryland is doing.

**Comment by nguerrera** [comment]
This is in PR, with the caveat that the task is not yet multi-targeted and so the in-box netfx tasks and targets will be used when using full framework msbuild, while the ported tasks and targets will be used when using dotnet build. I will file another issue to track the multi-targeting.

**Comment by nguerrera** [comment]
PR is merged. Waiting for new WPF build to insert into core-sdk. I will close this when that happens.

**Comment by nguerrera** [comment]
Correct. Tomorrow-ish, I will merge an updated wpf runtime and sdk. This will make `dotnet build` work using ported wpf tasks/targets that run on core. A temporary condition in the targets will still redirect to the classic inbox desktop targets in full framework msbuild / vs.

Next we'll multitarget the new build tasks and use them in VS and dotnet.

**Comment by onovotny** [comment]
When/what is the System.Security.Permissions issue pending on? (the one where the 4.0.2.0 one needs to be copied next to msbuild.exe)?

**Comment by nguerrera** [comment]
It is pending on these new ported targets. When `dotnet build` picks them up first, it won't have this problem. And when VS picks them up second after multitargeting, it won't either.

**Comment by onovotny** [comment]
Perfect. I guess I need VS to do it for real since I'm building the packaging project, and those targets are not in the .NET Core SDK.

Do you have any rough ETA when the multi-targeting be ready/merged? (not sure the level of effort, etc)

**Comment by nguerrera** [comment]
No ETA yet. But it’s a top priority.

**Comment by nguerrera** [comment]
This is done as of build 3.0.100-alpha1-009689.

Multi-targeting is tracked by #10174
